### DISCUSSION TITLE
Devhall.bsp

### DISCUSSION THREAD

**Discussion by mapkidd** [nzp-team/nzportable#1017] [Map Releases]
### --- DEVHALL ---
did you ever need a pretty basic 2-story house made of just developer textures with randomized perks and a PaP in the spawnroom?
probably not, but look no further; for i have made such a map!
### [---> D O W N L O A D <---](https://archive.org/download/Devhall/Devhall.zip)

![image](https://github.com/user-attachments/assets/afd895d7-afaa-459d-a856-8ce2d9bbf329)
**current issues:**

- camping on stairs is OP
- waypoint error spamming in console
- zombie spawn and barricade in the stairwell don't work
- zombies cannot yet enter the upstairs due to waypoint fuckery (i'm very new to mapping so i dunno how to fix it)

**Comment by spectrum1-wave** [comment]
Nice map but, what perks are included? ( Im too lazy to find them )

**Reply by Useless3** [reply]
Speed-Cola, Jug, Double Tap 2.0, Quick Revive, and Stamin-Up.
